internal/acctest/config/authenticationselector/authentication_selector_test.go

```diff
@@ -127,32 +124,27 @@ resource "pingfederate_authentication_selector" "%[1]s" {
         name  = "Add or Update AuthN Context Attribute"
         value = "%[4]s"
       },
-      {
-        name  = "Override AuthN Context for Flow"
```

Comment:
What's the restriction on this? We could follow the example of some other tests and add this in if it makes sense for the version being tested.

Reply:
added version check